passiveData/build.gradle.kts

```diff
@@ -25,28 +51,64 @@ kotlin {
     sourceSets {
         val commonMain by getting {
             dependencies {
+                implementation(Deps.Coroutines.core) {
+                    version {
+                        strictly(Versions.kotlinCoroutines)
+                    }
+                }
+
                 implementation(Deps.KotlinX.dateTime)
-                api (Deps.KotlinX.serializationJson)
+                api(Deps.KotlinX.serializationJson)
+
+                implementation(Deps.Ktor.clientCore)
+                //Is api to give depending modules access to JsonElement
+                api(Deps.Ktor.clientSerialization)
+                implementation(Deps.Ktor.clientLogging)
+
+                // koin
+                api(Deps.Koin.core)
+
+                implementation(Deps.Napier.napier)
+
+                implementation("com.google.android.gms:play-services-location:18.0.0")
+
             }
         }
         val commonTest by getting {
             dependencies {
                 implementation(kotlin("test-common"))
                 implementation(kotlin("test-annotations-common"))
+
+                implementation(Deps.Ktor.clientMock)
+
+                implementation(Deps.Ktor.clientMock)
+                api(Deps.Koin.test)
             }
         }
         val androidMain by getting {
             dependencies {
+                api(Deps.Coroutines.android)
+
                 api(Deps.AssessmentModel.sdk)
```

Comment:
@syoung-smallwisdom I added an AssessmentModel dependency for Android only. Can remove if you think that is better

Android is fine with transitive dependencies like
MTB -> AssessmentModels
         -> MobilePassiveData -> AssessmentModels
         
      I'm just using it alias in platform-specific implementations of classes like `StringEnum`

Reply:
My confusion is more around the interfaces that are defined in commonMain... so.... if I understand your response, you can have a circular reference? ie. AssessmentModel knowns about MobilePassiveData which knows about AssessmentModel? Magical!

Reply:
Darn. I was really hoping it could cuz that would be *SO* awesome. *sigh*

Would it make more sense and be less weird to go the other way?  I mean, right now iOS isn't using any of the Kotlin stuff except BridgeClientKMM b/c of assorted tech limitations.

MTB -> AssessmentModel -> MobilePassiveData

That's the direction on iOS

MTB -> SageResearch -> MobilePassiveData -> JsonModel